README.md

```diff
@@ -2,54 +2,56 @@
   ![Pegasys Orchestrate](static/orchestrate-logo.png)
 </p>
 
-# PegaSys Orchestrate Quick-Start
+# PegaSys Orchestrate Quickstart
 
 PegaSys Orchestrate is a platform that enables enterprises to easily build secure
 and reliable applications on Ethereum blockchains.
 
-It provides advanced features to connect to blockchain networks:
+Orchestrate provides advanced features when connected to blockchain networks:
 
-- Transaction management (transaction crafting, gas management, nonce management, transaction listening...)
+- Transaction management (transaction crafting, gas management, nonce management, and transaction listening)
 - Account management with private key storage in Hashicorp Vault
-- Smart Contract Registry
-- Multi-chain & Multi-protocol (public or private)
+- Smart contract registry
+- Public and private transactions 
+- Multi-chain.
 
-For more information please refer to [PegaSys Orchestrate official Documentation](https://docs.orchestrate.pegasys.tech/).
+For more information, refer to the [PegaSys Orchestrate official Documentation](https://docs.orchestrate.pegasys.tech/).
 
-During this quick-start you will manipulate:
+During this quick-start you use:
 
-- Orchestrate Command Line Interface (through npm scripts),
-- Orchestrate REST API using Curl,
+- Orchestrate command line interface using npm scripts.
```

Comment:
@MadelineMurray this is the famous list issue. I will never get used to it as the rules are so different in french.